wandb/apis/public/jobs.py

```diff
@@ -141,6 +143,11 @@ def _configure_launch_project_artifact(self, launch_project):
         else:
             launch_project.set_job_entry_point(self._entrypoint)
 
+        if self._dockerfile:
+            launch_project.set_job_dockerfile(self._dockerfile)
+        if self._build_context:
+            launch_project.set_job_build_context(self._build_context)
+
```

Comment:
Do we want to do this in the repo case too?

Reply:
good catch!